samples/led-matrix-weather/led-matrix-weather.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <RootNamespace>led_matrix_weather</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">

```

Comment:
Given that these are sample projects which we ideally want people to copy-paste and have them working, could we change these two references to package references instead?

Reply:
@joperezr this needs relatively recent version of the packages otherwise it won't display correctly. I.e.: https://github.com/dotnet/iot/pull/737
Also I'm planning to use this project for testing in the future